lms/celery.py

```diff
@@ -10,11 +10,15 @@
 
 from celery import Celery
 
+from openedx.core.lib.celery import assert_celery_uninstantiated
 from openedx.core.lib.celery.routers import AlternateEnvironmentRouter
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'lms.envs.production')
 
+# Prevent instantiation of both CMS and LMS celery.
+# See assert_celery_uninstantiated for more details.
+assert_celery_uninstantiated('lms')
```

Comment:
This ends up breaking tests, which load both cms and lms files and also don't define a `SERVICE_VARIANT` environment variable, thus loading both `cms.celery` and `lms.celery`. There are several possible options, but none totally straightforward, I think.